install_config/configuring_vsphere.adoc

```diff
@@ -235,3 +235,13 @@ Therefore, *_node-config.yaml_* must be in *_/etc/origin/node_* rather than
 [[vsphere-applying-configuration-changes]]
 == Applying Configuration Changes
 include::install_config/topics/applying_configuration_changes.adoc[]
+
+[[vsphere-backup]]
+== Backup of Persistent Volumes
+{product-title} provisions new volumes as *independent persistent disks* so
+it can freely attach and detach the volume any node in the cluster. As
+a consequence,
+link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vm_admin.doc/GUID-53F65726-A23B-4CF0-A7D5-48E584B88613.html[it is not possible to backup these volumes using snapshots].
+
+It's recommended to stop application that uses a PV, clone its persistent disk
+and start the application to take a backup of a volume.
```

Comment:
Suggest changing to "It is recommend to stop the application using the PV, clone its persistent disk, and restart the application to create a backup of the volume."

Reply:
s/using/that uses

Reply:
I added steps 4 and 5:

1. Stop the application using the PV.
2. Clone the persistent disk.
3. Restart the application.
4. Create a backup of the cloned disk.
5. Delete the cloned disk.